src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1737,7 +1737,8 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
                             ->arrayPrototype()
                                 ->children()
                                     ->scalarNode('lock')->defaultValue('lock.factory')->end()
-                                    ->scalarNode('storage')->defaultValue('cache.app')->end()
+                                    ->scalarNode('cache_pool')->defaultValue('cache.app')->end()
+                                    ->scalarNode('storage_service')->defaultNull()->end()
```

Comment:
We need validation to prevent both from being set.

And I would LOVE an `info()` on all of these

Reply:
`cache_pool` will always be set (as it has a default value). So I think we should just document that `storage_service` has precedence on `cache_pool`?

Reply:
Oh, duh, yes. Hmm, yea, that works!